pkg/simple/client/auditing/options.go

```diff
@@ -61,6 +64,22 @@ func (s *Options) AddFlags(fs *pflag.FlagSet, c *Options) {
 	fs.BoolVar(&s.Enable, "auditing-enabled", c.Enable, "Enable auditing component or not. ")
 
 	fs.StringVar(&s.WebhookUrl, "auditing-webhook-url", c.WebhookUrl, "Auditing wehook url")
+
+	fs.BoolVar(&s.BasicAuth, "auditing-elasticsearch-basicAuth", c.BasicAuth, ""+
+		"Elasticsearch auditing service basic auth enabled. KubeSphere is using elastic as auditing store, "+
+		"if this filed left blank, KubeSphere will use kubernetes builtin log API instead, and"+
```

Comment:
```suggestion
		"if set to true, KubeSphere will connect to ElasticSearch using provided username and password by auditing-elasticsearch-username and auditing-elasticsearch-username. Otherwise, KubeSphere will do an anonymous connection."+
```